src/models/RoutingSlip.ts

```diff
@@ -38,3 +38,11 @@ export interface LinkedRoutingSlips{
   children?: RoutingSlip[],
   parent?: RoutingSlip,
 }
+
+export interface ManualTransactionDetails{
+  quantity?: number,
+  referenceNumber?: string,
+  amount?: number,
+  isFutureEffectiveDate?: boolean,
```

Comment:
isFutureEffectiveDate or isFutureEffectiveFilng? Please ask sumesh whats the right term

Reply:
Confirmed with Sumesh and refactored it to isFutureEffectiveFiling. Thanks